app/audit/AuditService.scala

```diff
@@ -58,7 +61,11 @@ class AuditService @Inject() (auditConnector: AuditConnector)(implicit ec: Execu
           "sortCode" -> journey.bankDetails.sortCode
         ),
         "installments" -> Json.obj(
+          "initialPaymentAmount" -> journey.schedule.initialPayment,
           "installment" -> journey.schedule.instalments.getClass.toString,
```

Comment:
Can this 'installment' field be fixed so it puts in the serialised list of installments as json rather than the name of the class?

Reply:
Yeah I'll have a look into it. i thought it was a bit weird it was just returning the class as well.

Reply:
Actually I had a look at the ticket and in the example it has installment returning a type of class.